package.json

```diff
@@ -26,7 +25,6 @@
     "jquery-roving-tabindex": "^1.0.1",
     "jquery-tabs": "^0.5.3",
     "mark.js": "^8.11.1",
-    "rails-erb-loader": "^5.5.2",
```

Comment:
Do we want to also remove https://github.com/trade-tariff/trade-tariff-frontend/blob/main/config/webpack/loaders/erb.js

Reply:
good point - I'll amend that